career-readiness.md

```diff
@@ -10,14 +10,14 @@ There are lots of ways to create a portfolio. It is common to rework your portfo
 
 It is time to decide on your tech stack. We recommend either HTML/CSS or React.
 
-List your chosen tech stack here:
+List your chosen tech stack here: html/css
 
 ### LinkedIn
 
 Your LinkedIn url:
 
 Check the boxes below to certify that you have completed the following items on LinkedIn:
 
-- [ ] create a new post about your first three weeks at LEARN. Topics can include (but are not limited to) React, Ruby, projects you created, sharing a project you created, asking for advice, sharing advice.
-- [ ] add at least ten more professional connections. Connections can include your cohort mates, LEARN alumni, LEARN staff, following companies of interest, friends and family, and former coworkers.
-- [ ] react (like, celebrate, comment, etc) to ten posts from your existing connections.
+- [x] create a new post about your first three weeks at LEARN. Topics can include (but are not limited to) React, Ruby, projects you created, sharing a project you created, asking for advice, sharing advice.
+- [x] add at least ten more professional connections. Connections can include your cohort mates, LEARN alumni, LEARN staff, following companies of interest, friends and family, and former coworkers.
```

Comment:
Looks like you have 24 connections currently.  Let's work on getting this number a lot higher by next week.